docs/coverage/lcov-report/sorter.js

```diff
@@ -0,0 +1,158 @@
+var addSorting = (function () {
```

Comment:
Hm. Ok, just to clarify and make sure I understand correctly, the .html files above are essentially ways of displaying the coverage of the contracts. Is this file for helping with that task also?